pkg/machinery/config/types/v1alpha1/v1alpha1_provider.go

```diff
@@ -927,6 +927,10 @@ func (t *TimeConfig) Servers() []string {
 
 // Image implements the config.Provider interface.
 func (i *InstallConfig) Image() string {
+	if i.InstallImage == "" {
```

Comment:
I couldn't find anywhere in the code that might block us from even allowing this (validation of the install image for example). If I am missing something, please let me know.

Reply:
We could try keeping install image empty by default in talosctl cluster create (as a flag) to verify this

Reply:
Ah, good idea.